lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/IdentityVerificationService.java

```diff
@@ -80,9 +83,16 @@ public DocumentCheckVerificationResult verifyIdentity(DrivingPermitForm drivingP
             LOGGER.info("Form data validated");
             eventProbe.counterMetric(FORM_DATA_VALIDATION_PASS);
 
-            DocumentCheckResult documentCheckResult =
-                    thirdPartyGateway.performDocumentCheck(drivingPermitData);
+            DocumentCheckResult documentCheckResult = new DocumentCheckResult();
 
+            if (useDcs) {
+                documentCheckResult = thirdPartyGateway.performDocumentCheck(drivingPermitData);
+            } /*else {
```

Comment:
Dont think this needs commented out. Its just a condition that shouldnt be hit as part of the current flow

Reply:
Doesn't need to be just didn't want to bring in unused code, also want this to be clear for when we make the method that calls DVA directly

Reply:
But with it commented out we cant test the feature toggle which we want to do as part of this ticket. Its a pretty basic test but as were going to be relying on this toggle heavily going forward I think we should have it